source/quick-start/create-a-connection-string.txt

```diff
@@ -0,0 +1,65 @@
+.. _node-quick-start-connection-string:
+
+==========================
+Create a Connection String
+==========================
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and 
+port of your deployment, the authentication mechanism, user credentials 
+when applicable, and connection options.
+
+To connect to an instance or deployment not hosted on Atlas, see
+:ref:`Other Ways to Connect to MongoDB <node-other-ways-to-connect>`.
```

Comment:
Suggestion: 
Clicking on this link takes the user out of the tutorial completely, which also collapses it in the sidebar. I think this could be confusing to some and they may struggle to find their way back, or to find where they left off when they clicked it. Could we make this a link that opens in a new tab instead to mitigate those potential issues?

Reply:
Great suggestion! I think it might be ok this way for a couple reasons:

1) The quick start is designed to achieve the goal of connecting via Atlas and requires the sample data sets to be loaded. If they are connecting to a deployment not hosted on Atlas, the rest of the directions won't really work without a fair amount of sleuthing.
2) I'm not sure there's a way to specify a new tab with a `:ref:` or `:doc:` directive; the external links will be styled differently, and could be confusing since they're generally used for external links only. Not sure if there's any written guidance, but I'm guessing they're treated as two different entities in the UX design based on the appended icon that follows external links.